### ISSUE TITLE
Call for Ideas for Summer of Code 2018

### ISSUE THREAD

**Issue by bzz** [bblfsh/documentation#132]
Its GSoC 2018 org CFP period and I thought Bblfsh project might want to participate, so why do not we start gathering preliminary project ideas?

Some from the top of my head include i.e adding more drivers for  new languages.

What do you guys think?

**Comment by abeaumont** [comment]
Being realistic, I think the chances of being accepted is very low, so we should take into account before we want to invest time on this. That been said, the amount of work needed for the initial application is not too big. If we want to proceed, it'd be nice to know some things, even before we start gathering the project ideas:

- Who's going to drive the proposal?
- Who are the potential mentors?

So, I'd like to know what do you think about all this @bzz, @mcuadros and @juanjux

**Comment by juanjux** [comment]
- GSoC would be good publicity, and also a helping hand in language-analysis would be very nice to have.
- I agree with @abeaumont that we should be careful not to spent too much time with this.

I wouldn't have any problem with managing the proposal and/or being a mentor.

**Comment by bzz** [comment]
Indeed, as Bblfsh is a young project - seems like there might be not enough interest this year yet. 

It might be better to try to participate next year.

**Comment by mcuadros** [comment]
@campoy
